### PR TITLE
Nicolas/514 718 restore sentry in the console

### PR DESCRIPTION
**Pull request by callicles**
* Add conditional to only instrument cli in production
* Fix the dev version dependencies for the npm package so that we don't have `pnpm.lock` file updates all the time
* Remove Analytics that is only available on vercel
* Adds prettier to the pre-commit hook and an empty prettier config file as a reference for all tools

### CODE REVIEW

File: apps/moose-cli-npm/package.json [hunk 1]
```diff
@@ -23,9 +23,9 @@
     "typescript": "^4.9.5"
   },
   "optionalDependencies": {
-    "@514labs/moose-cli-darwin-arm64": "latest",
-    "@514labs/moose-cli-darwin-x64": "latest",
-    "@514labs/moose-cli-linux-arm64": "latest",
-    "@514labs/moose-cli-linux-x64": "latest"
+    "@514labs/moose-cli-darwin-arm64": "0.3.205",
```
Comment (cjus) [inline apps/moose-cli-npm/package.json:26]: Is the version set elsewhere? I expect this file and values would be updated by the CI/CD process?

File: apps/framework-cli/src/main.rs [hunk 2]
```diff
@@ -14,14 +14,16 @@ fn main() {
...
-        ("https://83941f36fe439ebe1ffd4d35b834ed7a@o4505851966128128.ingest.sentry.io/4505851967963136", 
-        sentry::ClientOptions {
-        release: sentry::release_name!(),
-        traces_sample_rate: 1.0,
-        environment: Some(envionment.into()),
-        ..Default::default()
-      }));
+    if envionment == "production" {
+        let _guard = sentry::init(
+            ("https://83941f36fe439ebe1ffd4d35b834ed7a@o4505851966128128.ingest.sentry.io/4505851967963136",
```
Comment (cjus) [inline apps/framework-cli/src/main.rs:19]: Is exposing this URL a security issue? Could a bad actor mess with our metric?

**Review by cjus — COMMENTED** [review]
Some questions but generally LGTM

File: apps/moose-cli-npm/package.json (hunk 1 above)
Comment (callicles) [reply apps/moose-cli-npm/package.json:26]: It is! this is mostly for the dev setup.

The script that bumps those versions during the release process can be found here: https://github.com/514-labs/moose/blob/main/apps/moose-cli-npm/scripts/release-cli.sh

File: apps/framework-cli/src/main.rs (hunk 2 above)
Comment (callicles) [reply apps/framework-cli/src/main.rs:19]: Yeah - I have been thinking about that. Unfortunatly this is software that runs anonymously on people's laptop. As such we need to be able to get metrics anonymously. We could make it a bit harder by embedding it at build time. But if someone is motivated, they will be able to get that back from the binary anyway.

**Review by cjus — APPROVED** [review]
Thanks for the replies. LGTM

**Pull request merged**